### ISSUE TITLE
Own cards are visible

### ISSUE THREAD

**Issue by yuki-93** [bstnfrmry/hanabi#277]
It happens from time to time, the my own cards become visible, so we can not proceed with the game. 

I don't figure out the reason yet, but I observed, that this happens for all my active games. So maybe a problem with the sessionId or data read from the device.

**Comment by benjamintd** [comment]
Can you try visiting https://hanab.cards/reset-session ? Let me know if it solves your problem.

**Comment by yuki-93** [comment]
Nope, the cards are still visible in the game. 

Does it help if I send you over the game-link?

**Comment by benjamintd** [comment]
Yes please, if you don't mind. It's probably device-specific though.

**Comment by yuki-93** [comment]
Here: https://www.hanab.cards/de/NearbyCraneCopper

I think not. I am having an iOS device but friends of me experience similar issues also with Android.

**Comment by benjamintd** [comment]
I meant depending on your session/localStorage, etc. 

I can reproduce the bug with your url, thanks.

**Comment by yuki-93** [comment]
@benjamintd any updates on this, if you are able to find a fix? :)

**Comment by yuki-93** [comment]
I am having the issue again. All my cards on all games become visible at the same time. Maybe s.th. regarding cookie lifetime?

Also I observed, I am not able to use the chat now, even as my move is next. 

Hopefully this helps a bit more.
